internal/pkg/deploy/cloudformation/stack/testdata/environments/template-with-basic-manifest.yml

```diff
@@ -993,6 +997,42 @@ Resources:
       DomainName: !Ref AppDNSName
       PublicAccessDNS: !GetAtt PublicLoadBalancer.DNSName
       PublicAccessHostedZone: !GetAtt PublicLoadBalancer.CanonicalHostedZoneID
+  AppRunnerVpcEndpointSecurityGroup:
+    Metadata:
+      'aws:copilot:description': 'A security group to services in this environment talk to App Runner private services'
```

Comment:
```suggestion
      'aws:copilot:description': 'A security group to allow services in this environment talk to App Runner private services'
```

Reply:
see what you think of https://github.com/aws/copilot-cli/pull/4111#discussion_r1005012535 😊